NYTimes.com.js

```diff
@@ -37,8 +37,18 @@
 
 
 function detectWeb(doc, url) {
-	if (getSearchResults(doc, true)) {
-		return "multiple";
+	//we use another function name to avoid confusions with the
```

Comment:
Yes, EM is exporting `detectWeb` see here https://github.com/zotero/translators/blob/master/Embedded%20Metadata.js#L813 . Maybe we should check whether we ever need that function in another call...? However, they should be sandboxed or in another scope, but that is not working correctly (there should be already an issue about this).